### PR TITLE
Minor cleanups to the specification

### PR DESCRIPTION
**Pull request by MarkLodato**

### CODE REVIEW

File: specification.md
Comment (joshuagl) [inline specification.md:83]: Once #11 is merged, we probably want to add something here about agreeing how to compute keyid?

Comment (mnm678) [reply specification.md:83]: As long as there aren't duplicate keyids, I don't think the mechanism for computing keyids needs to be known by the client.

Comment (joshuagl) [reply specification.md:83]: You're quite right, thanks Marina.

Comment (MarkLodato) [reply specification.md:83]: Yup, agreed. We'd want to say something like, "Agree on how keyid will be used." On the signer side, this means how it is computed; on the verifier side, it is how the key is looked up given the keyid. We should also explain that the verifier SHOULD try all possible keys that match that keyid.

Comment (adityasaky) [reply specification.md:83]: #11 has now been merged, apologies for the delay.

Comment (MarkLodato) [reply specification.md:94]: Thanks! No delay at all!

My inclination is to merge this as-is and leave further specification to #9. Currently this doesn't have any mention of multiple keys at all, and without multiple keys, keyid doesn't add any value.

Comment (adityasaky) [reply specification.md:94]: I agree, let's leave it to #9.

**Review by adityasaky — APPROVED** [review]
Thanks, @MarkLodato!

**Pull request merged**
